common/components/Header/components/TabsOptions.jsx

```diff
@@ -63,10 +61,12 @@ export default class TabsOptions extends Component {
                         <ul className='nav-inner'>
                             {
                                 tabs.map((object, i) => {
+                                        // if the window pathname is the same or similar to the tab objects name, set the active toggle
+                                        let activeOrNot = (window.location.pathname === object.name || window.location.pathname.substring(1) === object.name) ? 'active' : '';
```

Comment:
Ah! Sorry for being a stickler but this is a `const` situation :X

After that this is good to merge.